snippets/scripts/components-forge-field-to-compass-components/migrateCompassCFToComponent.py

```diff
@@ -205,7 +205,12 @@ def get_formatted_components(components):
 
 
 def update_issue(domain_name, user_name, api_token, issue_id, ari, existing_components, allowed_components_dict):
-    component_details = allowed_components_dict[ari]
+    try:
+        component_details = allowed_components_dict[ari]
+    except KeyError:
+        print(f"Couldn't copy issueId: {issue_id} issue's Compass custom field value to its Components field because of wrong component {ari}. "
+              f"Please try again later.")
+        return
```

Comment:
By Adding this change, instead of getting errors for components not found, we will log error and continue finish migration. At the same time, the performance seems got improved as well.
Just posted my local log result: 
![Screenshot 2024-06-12 at 15 00 23](https://github.com/atlassian-labs/compass-examples/assets/143025715/da9214b8-71d7-4eeb-baf5-335b2ff6eb94)